app/assets/stylesheets/application.scss

```diff
@@ -74,3 +103,13 @@ ul {
   margin-right: 5px;
   width: 50px;
 }
+
+#search {
+  width:200px;
```

Comment:
Properties should be ordered background-color, border-color, -moz-border-radius, -webkit-border-radius, border-radius, color, width<br>Colon after property should be followed by one space